star_tides/api/controllers/base_controller.py

```diff
@@ -91,4 +92,28 @@ def decode_basic_auth(auth_string):
 
         return user_id, password
 
+    @staticmethod
+    def decode_jwt() -> dict:
+        # TODO create exceptions for authorization
+        token = None
+        decoded_jwt = None
+
+        header = request.headers.get('Authorization')
+        if header is None:
+            raise Exception('No Authorization header present.')
+        try:
+            token = header.split(' ')[1]
```

Comment:
Nit: slightly more "readable" to do a regexp capture IMO. Another approach would be:

```python
header[len('Bearer '):]
```

or similar. Then you can get rid of the try/except.

Reply:
I'm trash so I don't know regex that well. I do think that it should stay in a try-except because people can send headers that could cause that to throw an exception.